Form_validation.php

```diff
@@ -0,0 +1,88 @@
+<?php 
+  $valid = array();
+  $errors = array();
+  if ($_POST) {
+
+    $title = $_POST['title'];
+
+    if (!$title) {
+      $errors['title'] = 'Името е задължително поле.';
+      echo $errors['title'];
+      } 
+      elseif (strlen($title) > 150) 
+      {
+        $errors['title'] = 'Името има максимална дължина 150 символа.';
+		echo $errors['title']; 
+      } 
+      else {
+        $valid['title'] = $title;   
+      }
+
+    $teacher = $_POST['teacher'];
+    if (!$teacher) {
+      $errors['teacher'] = 'Името на преподавател е задължително поле.';
+	  echo $errors['teacher'];   
+      } 
+      elseif (strlen($teacher) > 200) 
+      {
+        $errors['teacher'] = 'Името на преподавател има максимална дължина 200 символа.';
+        echo $errors['teacher'];       
+      } 
+      else {
+        $valid['teacher'] = $teacher;   
+      }
+
+      $description = $_POST['description'];
+      if (!$description) {
+        $errors['description'] = 'Описанието на дисциплината е задължително поле.';
+        echo $errors['description'];   
+      } 
+      elseif (strlen($description) < 10) 
+      {
+        $errors['description'] = 'Описанието на дисциплината има минимална дължина 10 символа.';
+        echo $errors['description'];  
+      } 
+      else {
+        $valid['description'] = $description;   
+      }
+
+      $group = $_POST['group'];
+      if(!$group) {
+        //it is filled with the first option by default, so can not be missing
+      }
+      else{
+        $valid['group'] = $group;
+      }
+
+      $credits = $_POST['credits'];
+      if($credits < 0) {
+        $errors['credits'] = 'Броят кредити трябва да е цяло положително число';
+        echo $errors['credits'];  
+      }
+      else{
+        $valid['credits'] = $credits;   
+      }
+	  
+	  if (count($valid) == 5) {
+                  $filename = 'data.txt';
+                  file_put_contents($filename, "Име на предмет: ", FILE_APPEND | LOCK_EX);
```

Comment:
When I submit the form, I receive no result on the screen. Was everything successful? Did something fail? What exactly failed? What do I need to do to fix it? How can I (as a dumb user not knowing about the "back" button of the browser) navigate to the previous page and try to fill the form once again? What was the exact information that was recorded by the server to the file?
Here is the acceptance criteria for the task: https://iliasky.com/www/presentations/Forms.html?read=1#11